shell/platform/windows/flutter_windows_view.cc

```diff
@@ -12,6 +12,9 @@
 
 namespace flutter {
 
+/// The ID of the root node in Flutter's semantics tree.
+inline constexpr int32_t kRootNodeId = 0;
```

Comment:
We should probably define this in some common place, I noticed I hard coded this in mac as well, my mistake

Reply:
maybe in shell/platform/common/accessibility_bridge.h

Reply:
Agreed -- we use this in a bunch of places. I'll do that as a pre-factoring patch then rebase this on that.

Reply:
Done in #29856.